src/components/ConfigurationRow.tsx

```diff
@@ -123,16 +145,16 @@ export const getConfigurationRow = ({
         })}
       >
         <div className="mono-font">
-          <b>{inheritedValue}</b>
+          <b>{readOnlyRenderer(metadata.value)}</b>
         </div>
-        {inheritedValue && (
+        {metadata && (
           <div className="p-text--small u-text--muted">
-            From: {inheritSource}
+            From: {metadata.source}
           </div>
         )}
       </div>
     ),
-    override: isReadOnly
+    override: formik.values.readOnly
```

Comment:
Unrelated to this PR, but the ternary below has the same indentation level which I find confusing since the last two conditions depends on the second condition (isOverriden). wdyt? Is that part of the general style guide?

Reply:
I'll change it to a function with early exits, which should be much easier to read.